subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AbstractRealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -28,7 +28,7 @@ class AbstractRealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionPe
 
     def setup() {
         runner.args = [AndroidGradlePluginVersions.OVERRIDE_VERSION_CHECK]
-        runner.targetVersions = ["6.8-20201116230039+0000"]
+        runner.targetVersions = ["6.8-20201223103229+0000"]
```

Comment:
Why change this?

Reply:
@blindpirate [reverted my revert](https://github.com/gradle/gradle/pull/15738/commits/18f3930a270005177543e86e1e8f7c32edd352f7) from `release` on `master`, and those changes were part of the revert of the revert of the revert. I wouldn't touch it now and instead rebaseline `master` after these changes have been merged to lock in the performance improvements.